gammapy/irf/effective_area_table.py

```diff
@@ -311,3 +316,301 @@ def plot_area_vs_energy(self, filename=None, show_save_energy=True):
         if filename is not None:
             plt.savefig(filename)
             log.info('Wrote {0}'.format(filename))
+
+
+class OffsetDependentTableEffectiveArea(object):
+    """Offset-dependent radially-symmetric table effective area (``GCTAAeff2D FITS`` format).
+
+    The table is  read by interpolated by triangulating the input data with Qhull (http://www.qhull.org/R42).
```

Comment:
Refer to the scipy classes you are using from the docstring instead of mentioning Qhull here. E.g.
```
Interpolation is done with `~scipy.interpolate.RectBivariateSpline` or ...
```